Initial.py

```diff
@@ -12,4 +13,4 @@ def lambda_handler(event, context):
 
     print(f"Found {len(response['Reservations']['Instances'])} instances")
```

Comment:
The print method should refer to the len(instanceList) instead of {len(response['Reservations']['Instances'])}